MimeKit/Cryptography/SqlCertificateDatabase.cs

```diff
@@ -229,32 +229,12 @@ static DataTable CreateCrlsDataTable (string tableName)
 		/// <param name="column">The column to add.</param>
 		protected abstract void AddTableColumn (DbConnection connection, DataTable table, DataColumn column);
 
-		static string GetIndexName (string tableName, string[] columnNames)
+		public static string GetIndexName (string tableName, string[] columnNames)
 		{
```

Comment:
Does this need to be `public` or can it be `protected`?

Reply:
switched from public. no reason for it to be static afaik.